src/PioneerUtils.jsm

```diff
@@ -50,6 +52,11 @@ class PioneerUtils {
     return id;
   }
 
+  async isUserOptedIn() {
+    const addon = await AddonManager.getAddonByID("pioneer-opt-in@mozilla.org");
+    return addon !== null;
```

Comment:
This will return true for an add-on that is installed but disabled. We should check that too.